sdk/python/feast/infra/online_stores/helpers.py

```diff
@@ -62,3 +63,10 @@ def compute_entity_id(
             entity_key_serialization_version=entity_key_serialization_version,
         )
     ).hex()
+
+
+def _to_naive_utc(ts: datetime) -> datetime:
```

Comment:
would it make sense to set a defalt timezone instead of a none?

Reply:
not sure, i didn't implement it but just moved it out to helper function. it currently in singlestore module and imported into postgres module lol